### PR TITLE
[#176013578] Refactor pipeline

### PR DESCRIPTION
**Pull request by gquadrati**
- [x] switch from release-it to changelog
- [x] add code-review pipeline
- [x] setup code-review pipeline in azure 
- [x] test code-review pipeline
- [x] remove old azure pipeline
- [x] remove circleci
- [x] add deploy pipeline
- [x] setup deploy pipeline in azure
- [x] check service account and variables ( @balanza )
- [x] test deploy pipeline (@balanza )
- [x] Added SDK client generation step
- [x] Add NPM_CONNECTION variable in Azure DevOps
- [ ] Test Client Generation

### CODE REVIEW

**Review by balanza — APPROVED** [review]

**Comment by balanza** [comment]
I'll review accounts and set this up asap

**Review by pasqualedevita — CHANGES_REQUESTED** [review]
There is no staging slot

**Comment by balanza** [comment]
> There is no staging slot

Well spotted pasquale, thanks. Self-memento: always remember to double check with [infra repo](https://github.com/pagopa/io-infrastructure-live-new/)

**Comment by gquadrati** [comment]
> There is no staging slot

Sorry, I'll change it

File: .devops/deploy-pipelines.yml [hunk 1]
```diff
@@ -124,49 +124,25 @@ stages:
                 !CODEOWNERS
                 !__*/**/*
             displayName: 'Copy deploy files'
```
Comment (balanza) [inline .devops/deploy-pipelines.yml:126]: You need to publish the artifact, otherwise files get lost with the job

**Review by balanza — CHANGES_REQUESTED** [review]

File: .devops/deploy-pipelines.yml (hunk 1 above)
Comment (gquadrati) [reply .devops/deploy-pipelines.yml:126]: right, my bad.. I fixed it

File: .devops/deploy-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+                !CODEOWNERS
+                !__*/**/*
+            displayName: 'Copy deploy files'
+
+          - publish: $(System.DefaultWorkingDirectory)/bundle
+            artifact: Bundle
+
+  - stage: Deploy_production
+    dependsOn:
+      - Release
```
Comment (balanza) [inline .devops/deploy-pipelines.yml:133]:
```suggestion
      - Prepare_artifact
```

**Review by balanza — CHANGES_REQUESTED** [review]

**Review by balanza — APPROVED** [review]

**Review by pasqualedevita — APPROVED** [review]
Looks good, thanks @gquadrati @balanza

**Review by balanza — APPROVED** [review]

**Pull request merged**